android/src/main/java/com/horcrux/svg/TextView.java

```diff
@@ -20,7 +20,6 @@
 import android.view.ViewParent;
 import com.facebook.react.bridge.Dynamic;
 import com.facebook.react.bridge.ReactContext;
-import com.facebook.react.bridge.ReadableArray;
```

Comment:
Why do we have does prettier changes in the PR? I would remove them since they are not in the scope of it.

Reply:
restored.